### DISCUSSION TITLE
Add Rotate(X), Rotate(Y) and backface-visibility properties

### DISCUSSION THREAD

**Discussion by AbobkerElaghel** [tailwindlabs/tailwindcss#7850] [Ideas]
Please add the Rotate(X), Rotate(Y) and backface-visibility properties to the library, it does help in flipping elements 180deg on the y or x axis.

**Comment by sambauers** [comment]
I made this recently. It’s early days, but I think it has potential to fill this gap and others around 3D transforms.

https://github.com/sambauers/tailwindcss-3d

A bit lacking in documentation, but should make sense to most folks familiar with Tailwind transforms.
